movement/movement.c

```diff
@@ -218,6 +218,26 @@ void movement_illuminate_led(void) {
     }
 }
 
+bool movement_default_loop_handler(movement_event_t event, movement_settings_t *settings) {
+    (void)settings;
+
+    switch (event.event_type) {
+        case EVENT_MODE_BUTTON_UP:
+            movement_move_to_next_face();
+            break;
+        case EVENT_LIGHT_BUTTON_DOWN:
+            movement_illuminate_led();
```

Comment:
I think this could be confusing, because both this function and the previous function require the watch _not_ to sleep, but this is handled elsewhere. Maybe just a comment explaining why it's 'safe' to return true here, and amend the comment in `template.c` (see below).

Reply:
Actually I think it was always a mistake for the watch to return false in these cases; `movement_move_to_next_face` sets a variable indicating that the watch face changed (which suppresses standby for one cycle) and `movement_illuminate_led`also sets a boolean that keeps the watch awake while the PWM is in operation. The `return false` in some watch faces was harmless (and confusing) but ultimately not necessary.

Reply:
Sorry, I should have been clearer - that was my point, that 'sleeping' is not necessary because it's handled. But then in the comment in template it says that if you mess with the LED you _do_ need to return false, which isn't true.

Err, I'm not making this better, am I?

Suggestion:

Comment here to say '// Safe to return true even though we don't want to sleep because movement forces us not to sleep'

And change template.c comment to mention that it's not necessary to return false in the LED case or the buzzer case as this is handled automatically.

Reply:
Aha, I get your point. I think the issue is that the documentation around messing with the LED is inartfully worded. The point I was trying to make is that if you're using the low-level watch library functions to interact with the LED or buzzer, you have to return false. Calling the higher-level Movement functions for interacting with the LED or buzzer does not require anything special.

I will resolve this by clarifying that note.